product_docs/docs/biganimal/release/free_trial/detail/create_a_cluster/create_cluster_cli.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Creating a cluster on the commmand line
+navTitle: "Using the command line" 
+indexCards: none
+---
+
+
+We'll be using the [BigAnimal command line interface](/biganimal/latest/reference/cli/), which is a convenient wrapper to the [BigAnimal API](/biganimal/latest/reference/api/). To start, download [the latest binary](/biganimal/latest/reference/cli/) and move it to wherever your system will find executable files. If you have a Unix-style shell, your own `bin` directory is usually a good location.
+```shell
+mv biganimal $HOME/bin
+```
+
+It may also be necessary to give the executable proper permission:
+```shell
+chmod +x $HOME/bin/biganimal
+```
+
+
+Next, we need to create a credential on BigAnimal. You can pick any username you prefer.
+```shell
+biganimal create-credential -a portal.biganimal.com -o 443 -n newuser
+```
+
+The command will direct you to open a webpage and copy the randomly-generated, one-time code. You’ll need to log in (or already be logged in) to activate the credentials. You can see which credentials you have verified on the command line.
+
+```shell
+biganimal show-credentials
+```
+
+!!! caution
+If you add another credential, you’ll need to add `--credential [newuser]` to the following commands. If you only have one, the option is not needed. You may remove unneeded credentials using `biganimal delete-credential [olduser]`. Most users do not need to add multiple credentials.
+!!!
+
+In the free trial, the range of cluster options is somewhat limited, but should be more than sufficient for demonstrating the capabilities of BigAnimal.
+
+Pick the region closest to you from the list provided by biganimal:
+```shell
+biganimal show-regions
+```
+
+Other options can be queried via the CLI, but the only variable we need to set for this demo is .
+
+Edit a new file called create_cluster.yaml:
+```yaml
+---
+clusterName: test_cluster # Pick a more meaningful name!
+password: mygoodpassword  # Please change this password!
+postgresType: epas
+postgresVersion: "14"
+provider: azure
+region: eastus2  # Select from the options given by show-regions
+instanceType: azure:Standard_E2s_v3
+volumeProperties: P1
+volumeType: azurepremiumstorage
+networking: public
+highAvailability: false
+---
+```
+
+Use the config file to create a new cluster:
+```shell
+biganimal create-cluster --clusterConfigFile create_cluster.yaml
+```
+
+If successful, the command will give you a command to check the status of your new cluster:
```

Comment:
I think this is supposed to say the command will give you "an id" not "a command"

Reply:
It gives you both - an Id and a command that uses the Id. This sentence is awkward though; will rephrase.